model/custom_status.go

```diff
@@ -30,10 +30,11 @@ var validCustomStatusDuration = map[string]bool{
 }
 
 type CustomStatus struct {
-	Emoji     string    `json:"emoji"`
-	Text      string    `json:"text"`
-	Duration  string    `json:"duration"`
-	ExpiresAt time.Time `json:"expires_at"`
+	Emoji        string    `json:"emoji"`
+	Text         string    `json:"text"`
+	Duration     string    `json:"duration"`
```

Comment:
Why is Duration a string? wouldn't it be better a [Duration from time](https://pkg.go.dev/time#Duration)?